package.json

```diff
@@ -69,7 +69,17 @@
     "typography-plugin-code": "^0.16.20"
   },
   "overrides": {
+    "@enact/docs-utils": {
```

Comment:
my npm audit command ran locally on develop branch showed only 2 sources of high/critical vulnerabilities
1. lodash.template - whch cannot be resolved until gatsby-plugin-offline updates it's source code
2 "path-to-regexp":, which can be solved with an override
![image](https://github.com/user-attachments/assets/dcdad0f7-effc-4804-b24a-23869286818c)

Can you please double check?